web/src/App.svelte

```diff
@@ -91,6 +91,25 @@
     mapDiv.innerHTML = "";
     mapDiv.appendChild($mapContents);
   }
+
+  async function getStyle(
+    basemap: string,
+  ): Promise<StyleSpecification | string> {
+    if (basemap != "streets-v2") {
+      return `https://api.maptiler.com/maps/${basemap}/style.json?key=${maptilerApiKey}`;
+    }
+
+    // streets-v2 uses a fill-extrusion layer for 3D buildings that's very distracting. Remove it, and make the regular buildings layer display at high zoom instead.
```

Comment:
I thought about downloading this JSON, modifying it manually, and storing in git, but for these two simple changes, that feels wrong. The basemap styles don't have breaking changes without a version bump.